bootstrapvz/common/tasks/grub.py

```diff
@@ -26,11 +26,27 @@ def run(cls, info):
 		from bootstrapvz.common.tools import sed_i
 		grub_def = os.path.join(info.root, 'etc/default/grub')
 		sed_i(grub_def, '^#GRUB_TERMINAL=console', 'GRUB_TERMINAL=console')
-		sed_i(grub_def, '^GRUB_CMDLINE_LINUX_DEFAULT="quiet"',
-		                'GRUB_CMDLINE_LINUX_DEFAULT="console=hvc0"')
-		sed_i(grub_def, '^GRUB_TIMEOUT=[0-9]+', 'GRUB_TIMEOUT=0\n'
-		                                        'GRUB_HIDDEN_TIMEOUT=0\n'
-		                                        'GRUB_HIDDEN_TIMEOUT_QUIET=true')
+		
+		# optionally, enable grub menu
+		if info.manifest.system.get('grub_enable_timeout') is True:
+			# print "grub: enabling menu"
+			sed_i(grub_def, '^GRUB_TIMEOUT=[0-9]+', 'GRUB_TIMEOUT=5')
+		else:
+			sed_i(grub_def, '^GRUB_TIMEOUT=[0-9]+', 'GRUB_TIMEOUT=0\n'
+													'GRUB_HIDDEN_TIMEOUT=0\n'
+													'GRUB_HIDDEN_TIMEOUT_QUIET=true')
+		# optionally, disable persistent network 
+		if info.manifest.system.get('grub_disable_pnin') is True:
+			# print "grub: disabling persistent network interface names"
+			sed_i(grub_def, '^GRUB_CMDLINE_LINUX_DEFAULT="quiet"',
+							'GRUB_CMDLINE_LINUX_DEFAULT="net.ifnames=0 biosdevname=0"')
+			sed_i(grub_def, '^GRUB_CMDLINE_LINUX=""',
+							'GRUB_CMDLINE_LINUX="net.ifnames=0 biosdevname=0"')
```

Comment:
This has to be done two times, for both `GRUB_CMDLINE_LINUX_DEFAULT` and `GRUB_CMDLINE_LINUX`?

Also, in this case, `console=hvc0` isn't needed anymore?